app/swagger/schemas/gibct/institution_programs.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Schemas
+    module Gibct
+      class InstitutionPrograms
+        include Swagger::Blocks
+
+        swagger_schema :GibctInstitutionProgramsAutocomplete do
+          key :required, %i[data meta links]
+
+          property :data, type: :array, minItems: 0, uniqueItems: true do
+            items do
+              property :id, type: :integer
+              property :value, type: :string
+              property :label, type: :string
+            end
+          end
+
+          property :meta, '$ref': :GibctInstitutionProgramsAutocompleteMeta
+
+          property :links, type: :object do
+            property :self, type: :string
+          end
+        end
+
+        swagger_schema :GibctInstitutionProgramsSearch do
+          key :required, %i[data meta links]
+
+          property :data, type: :array, maxItems: 10, uniqueItems: true do
+            items do
+              key :type, :object
+              key :required, %i[id type attributes]
+
+              property :id, type: :string
+              property :type, type: :string, enum: ['institution_programs']
+              property :attributes do
+                key :required, %i[facility_code description]
+
+                property :program_type, type: %i[null string],
+                                        enum: %w[IHL NCD OJT FLGT CORR]
+                property :description, type: :string
+                property :length_in_hours, type: %i[null string]
+                property :length_in_weeks, type: %i[null integer]
+                property :facility_code, type: :string
+                property :institution_name, type: :string
+                property :city, type: %i[null string]
+                property :state, type: %i[null string]
+                property :country, type: %i[null string]
+                property :preferred_provider, type: %i[null boolean]
+                property :tuition_amount, type: %i[null integer]
+                property :va_bah, type: %i[null number]
+                property :dod_bah, type: %i[null integer]
+              end
+            end
+          end
+
+          property :meta, '$ref': :GibctInstitutionProgramsSearchMeta
+          property :links, '$ref': :GibctInstitutionProgramsSearchLinks
+        end
+
+        swagger_schema :GibctInstitutionProgramsAutocompleteMeta do
+          key :type, :object
+          key :required, %i[version term]
+
+          property :version, '$ref': :GibctVersion
+          property :term, type: :string
+        end
+
+        swagger_schema :GibctInstitutionProgramsSearchMeta do
+          key :type, :object
+          key :required, %i[version count facets]
+
+          property :version, '$ref': :GibctVersion
+
+          property :count, type: :integer
+          property :facets, type: :object do
+            key :required, %i[type state country]
+            property :type, type: :object do
```

Comment:
Can we handle this in a follow up story? The existing GIDS services are also not well documented, and it seems like we could have one story to bring that up to spec without holding up approval for this PR (especially since it's not currently being used by vets-website in prod)

Reply:
these sort of things tend to be forgotten.  I'll approve if you link to an issue.

Reply:
@annaswims can you take a look at this so we can get this in?  Thanks!